Importer/Log/CsvLog.php

```diff
@@ -50,9 +50,12 @@ public function addDataError($index, $errorMessage)
     /**
      * {@inheritdoc}
      */
-    public function addCriticalError($errorMessage)
+    public function addCriticalError($message, $userError = false)
```

Comment:
Well the only reason I am not a big fan of this approach is because I do not see any technical reason for critical errors, when we abort manually.

So when the application crushes we have:
- `errorReason` but no `userMessage`

But when we add manually an application message we have:
- `userMessage` but no `errorReason`